src/V3AstNodeDType.h

```diff
@@ -53,7 +53,7 @@ class AstNodeDType VL_NOT_FINAL : public AstNode {
     }
 
 public:
-    ASTNODE_BASE_FUNCS(NodeDType)
+    ASTGEN_MEMBERS_NodeDType;
```

Comment:
```suggestion
    ASTGEN_MEMBERS;
```

So one less thing to mess up - astgen would take class name from the "class" line earlier (as it parses already)

Reply:
Note that astgen generates different macros for each type, so the actual text replacement is done by the C++ pre-processor (and not astgen itself). This is to allow IDEs to resolve to the canonical sources for symbol definitions. (I mentioned this earlier, regarding the input file also being the canonical source file, but augmented by generating macro definitions.) Hence each macro needs to be unique for the type it's inserted into. Also, you can't mess this up because astgen now checks it and it will error if you omit it or mistype it (`class AstFoo` must contain `ASTGEN_MEMBERS_Foo;`)